src/app/hearing-part/components/hearing-parts-preview/hearing-parts-preview.component.ts

```diff
@@ -62,11 +62,22 @@ export class HearingPartsPreviewComponent implements OnInit, OnChanges {
                     return moment.duration(item[property]).asMilliseconds();
 
                 case 'reservedJudge':
-                    return item[property] === undefined ? null : item[property].name;
+                    return this.getPropertyMemberOrNull(item, property, 'name');
+
+                case 'caseType':
+                case 'hearingType':
+                    return this.getPropertyMemberOrNull(item, property, 'description');
 
                 case 'priority':
                     return priorityValue(item[property]);
 
+                case 'scheduleStart':
+                case 'scheduleEnd':
+                    return (item[property] === undefined || item[property] === null) ? null : item[property].unix();
```

Comment:
minor why just not use falsey values ie.. 
return item[property] ? null : ....
https://j11y.io/javascript/truthy-falsey/

Reply:
Interesting first time I see this